dwave/system/testing.py

```diff
@@ -301,6 +450,12 @@ def sample(self, bqm, **kwargs):
             ss = ss.truncate(max_answers)
         
         return ss
+
+    def to_networkx_graph(self):
+        return _dnx_graph(self.properties['topology']['type'],
+                          self.properties['topology']['shape'],
+                          self.nodelist, self.edgelist)
```

Comment:
```suggestion
    to_networkx_graph = DWaveSampler.to_networkx_graph
```

Reply:
Forgot that I could inherit the method from DWaveSampler that way, its the best choice.

Reply:
If you do it like this, than calling it like this:
```
solver_graph = self.to_networkx_graph()
```
will do the right thing.

Reply:
In the end, I thought it tidier to define a separate public function. It has use cases independent of any particular sampler. It should probably be part of dwave_networkx, but it is located in dwave_sampler.py with the current edit. 
Happy to have this reverted and inherited by some other pattern if preferred.